server/chat-plugins/scavengers.js

```diff
@@ -1130,6 +1132,25 @@ let commands = {
 		}
 	},
 
+	addhint: 'spoiler',
+	spoiler(target, room, user) {
+		if (!room.game || !room.game.scavGame) return this.errorReply(`There is no scavenger game currently running.`);
+		let game = room.game.childGame || room.game;
+		if ((!game.hosts.some(h => h.userid === user.userid) || !user.can('broadcast', null, room)) && game.staffHostId !== user.userid) return this.errorReply("You cannot add more hints if you are not the host.");
+
+		let elapsedTime = Date.now() - game.startTime;
+		if (elapsedTime < 600000 /* 10 minutes */) return this.errorReply("You can only use this command 10 minutes after the hunt starts.");
+
+		let [question, ...hint] = target.split(',');
+		question = parseInt(question) - 1;
+
+		if (!game.questions[question]) return this.errorReply(`Invalid question number.`);
+		game.questions[question].spoilers.push(hint.join(','));
+
+		room.addByUser(user, `Question #${question + 1} hint - spoiler: ${hint.join(',')}`);
```

Comment:
It may be a good idea to disallow empty hints. A command like `/scav spoiler 1` will send an empty spoiler to the room, which isn't really useful.